app/api/video_recordings.py

```diff
@@ -0,0 +1,110 @@
+from datetime import datetime
+
+from flask_rest_jsonapi import ResourceDetail, ResourceList
+from flask_rest_jsonapi.resource import ResourceRelationship
+
+from app.api.helpers.db import get_or_create, safe_query_kwargs
+from app.api.helpers.errors import ForbiddenError
+from app.api.helpers.permission_manager import has_access
+from app.api.helpers.permissions import jwt_required
+from app.api.schema.video_recordings import VideoRecordingSchema
+from app.api.video_channels.bbb import BigBlueButton
+from app.models import db
+from app.models.video_recording import VideoRecording
+from app.models.video_stream import VideoStream
+
+
+class VideoRecordingList(ResourceList):
+    def before_get(self, args, kwargs):
+        stream = safe_query_kwargs(VideoStream, kwargs, 'video_stream_id', 'id')
+        if not has_access('is_organizer', event_id=stream.event_id):
+            raise ForbiddenError(
+                {'pointer': 'event_id'},
+                'You need to be the event organizer to access video recordings.',
+            )
+
+        params = dict(
+            meetingID=stream.extra['response']['meetingID'],
+        )
+        channel = stream.channel
+        bbb = BigBlueButton(channel.api_url, channel.api_key)
+        result = bbb.request('getRecordings', params)
+
+        if result.data['response']['recordings']:
+            recordings = []
+            if type(result.data['response']['recordings']['recording']) is list:
+                recordings = result.data['response']['recordings']['recording']
+            else:
+                recordings.append(result.data['response']['recordings']['recording'])
```

Comment:
@sachinchauhan2889 why are we checking for list here, I am not understanding this?